SECURITY.md

```diff
@@ -0,0 +1,11 @@
+# Security Policy 
```

Comment:
This PR looks great to me! Just have one small comment about this file. WildFly recently tweaked it's SECURITY.md file so I think we can copy the new content from there:

https://github.com/wildfly/wildfly/blob/main/SECURITY.md

Reply:
SECURITY.md has been updated according the the current content of WildFly.